src/components/Tooltip.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  // parent block has to have position: relative styles
+  import { fly, fade } from 'svelte/transition';
+  import { circIn } from 'svelte/easing';
+  import { onMount } from 'svelte';
+
+  export let text = '';
+
+  const defaultTooltipInset = {
```

Comment:
I'm not exactly sure, but if it's a default set, how I can override it?
If I can't and it's hardcoded, I thin it could be named differently or just used inline